### ISSUE TITLE
pytorch 1.5 much slower than 1.4

### ISSUE THREAD

**Issue by mravanelli** [speechbrain/speechbrain#54]
I did some first experiments, and apparently the issue could be connected to jit. In fact, the significant slow down happens when using our custom rnn called ligru (that is the only module we compile):
model      |      pytorch1.5         |    pytorch1.4
ligru-jit           2 min 51 sec            1min 52 sec
ligru-nojit       3 min 32 sec            3min 45 sec

In practice, for some reason, jit is much more effective in pytorch 1.4 rather than pytorch 1.5 (at least on our ligru). I tried to simplify the model (e.g, removing batch norm, dropout, bidirectional, etc) and the issue still appears.
